MediaBrowser.Controller/LibraryTaskScheduler/LimitedConcurrencyLibraryScheduler.cs

```diff
@@ -96,14 +96,33 @@ async Task RunCleanupTask()
         }
     }
 
+    private bool ShouldForceSequencialOperation()
+    {
+        // if the user either set the setting to 1 or its unset and we we have less then 4 cores its better to run sequencially.
+        var fanoutSetting = _serverConfigurationManager.Configuration.LibraryScanFanoutConcurrency;
+        return fanoutSetting is 1 || (fanoutSetting <= 0 && Environment.ProcessorCount <= 3);
```

Comment:
```suggestion
        return fanoutSetting == 1 || (fanoutSetting <= 0 && Environment.ProcessorCount <= 3);
```

Compiles to the same thing but I think explicit is better